src/nav/AppContainer.js

```diff
@@ -66,6 +67,8 @@ class AppContainer extends PureComponent {
       actions.fetchEssentialInitialData();
       actions.fetchRestOfInitialData(pushToken);
       actions.fetchEvents();
+      actions.fetchUsersStatus(); // do a initial fetch
+      setInterval(() => actions.fetchUsersStatus(), config.activePingInterval);
```

Comment:
@Sam1301 Updated. Now using `state.app.isActive`. BTW when app will pause it will also be paused, so never wrong status will be send to server.

@borisyankov yeah, we need to request manually. Presence event is only received when someone becomes active. So current situation is 
- on app start presence is empty, (status is null thus no indicator is shown)
- as new users comes online, there presence gets stored in state (and green indicator is shown).
- Now if someone becomes idle or inactive there is no event (thus it is keep on showing green until app is restarted.)

On sending a ping focus, it will make sure self status is correct and in response we will get updated presence of all users.